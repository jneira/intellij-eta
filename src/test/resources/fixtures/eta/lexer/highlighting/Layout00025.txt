ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('Layout00025')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITimport ('import')
WHITE_SPACE ('           ')
ITconid ('Text.Parsec')
WHITE_SPACE ('\n')
ITimport ('import')
WHITE_SPACE ('           ')
ITconid ('Text.Parsec.String')
WHITE_SPACE ('\n\n')
ITvarid ('testParser')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
ITconid ('Parser')
WHITE_SPACE (' ')
ITconid ('Char')
WHITE_SPACE ('\n')
ITvarid ('testParser')
WHITE_SPACE (' ')
ITequal ('=')
WHITE_SPACE (' ')
ITdo ('do')
WHITE_SPACE ('\n    ')
ITvarid ('c')
WHITE_SPACE (' ')
ITlarrow ('<-')
WHITE_SPACE (' ')
ITvarid ('char')
WHITE_SPACE (' ')
ITchar (''c'')
WHITE_SPACE ('\n    ')
ITvarsym ('<?>')
WHITE_SPACE (' ')
ITstring ('"')
ITstring ('char c expected')
ITstring ('"')
WHITE_SPACE ('\n')
