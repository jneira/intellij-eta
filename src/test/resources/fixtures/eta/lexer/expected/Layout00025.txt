ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('Layout00025')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITvocurly ('')
ITimport ('import')
WHITE_SPACE ('           ')
ITqconid ('Text.Parsec')
WHITE_SPACE ('\n')
ITsemi ('')
ITimport ('import')
WHITE_SPACE ('           ')
ITqconid ('Text.Parsec.String')
WHITE_SPACE ('\n\n')
ITsemi ('')
ITvarid ('testParser')
WHITE_SPACE (' ')
ITdcolon ('::')
WHITE_SPACE (' ')
ITconid ('Parser')
WHITE_SPACE (' ')
ITconid ('Char')
WHITE_SPACE ('\n')
ITsemi ('')
ITvarid ('testParser')
WHITE_SPACE (' ')
ITequal ('=')
WHITE_SPACE (' ')
ITdo ('do')
WHITE_SPACE ('\n    ')
ITvocurly ('')
ITvarid ('c')
WHITE_SPACE (' ')
ITlarrow ('<-')
WHITE_SPACE (' ')
ITvarid ('char')
WHITE_SPACE (' ')
ITchar (''c'')
WHITE_SPACE ('\n    ')
ITsemi ('')
ITvarsym ('<?>')
WHITE_SPACE (' ')
ITstring ('"char c expected"')
WHITE_SPACE ('\n')
ITvccurly ('')
ITsemi ('')